### PR TITLE
[DOC][MINOR] Add shell interpreter docs to _navigation.html

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
After #1087 merged, a new docs `shell.md` was added. But in the docs website, still Shell interpreter link points to `pleasecontribute.html`. So I changed this link, applied TOC and added more descriptions. 

### What type of PR is it?
Documentation

### Todos
* [x] - Change `pleasecontribute.html` -> `shell.html`
* [x] - Apply TOC(table of contents)
* [x] - Add more description to `shell.md`

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no 
* Does this needs documentation? no

### CODE REVIEW

File: docs/interpreter/shell.md [hunk 1]
```diff
@@ -2,19 +2,39 @@
...
-Shell interpreter uses [Apache Commons Exec](https://commons.apache.org/proper/commons-exec) to execute external processes. 
+<div id="toc"></div>
 
+## Overview
+Shell interpreter uses [Apache Commons Exec](https://commons.apache.org/proper/commons-exec) to execute external processes. 
 In Zeppelin notebook, you can use ` %sh ` in the beginning of a paragraph to invoke system shell and run commands.
-Note: Currently each command runs as Zeppelin user.
 
-### Example
+> **Note :** Currently each command runs as Zeppelin user.
```
Comment (felixcheung) [inline docs/interpreter/shell.md]: perhaps it could help to explain "Zeppelin user" as "the user Zeppelin server is running as" (or similar)?

**Comment by felixcheung** [comment]
LGTM

**Comment by fvaleri** [comment]
@felixcheung agree.
@AhyoungRyu thanks for the integration.

**Comment by AhyoungRyu** [comment]
@felixcheung I updated `shell.md` as you suggested :)
I also applied TOC(Table of Contents) to `rest-credential.md` which is from #1030.

**Comment by felixcheung** [comment]
LGTM

**Pull request closed without merging**